
gsoap changelog for 2.8.13 states:
Deprecated old-style C++ service proxies and objects (use soapcpp2 -z1 flag to generate).

And VirtualBox needs those. so lets add -z1

Signed-off-by: Thomas Backlund <tmb@mageia.org>

diff -Nurp VirtualBox-4.3.0.p15/src/VBox/Main/webservice/Makefile.kmk VirtualBox-4.3.0.p16/src/VBox/Main/webservice/Makefile.kmk
--- VirtualBox-4.3.0.p15/src/VBox/Main/webservice/Makefile.kmk	2013-10-15 14:13:13.000000000 +0300
+++ VirtualBox-4.3.0.p16/src/VBox/Main/webservice/Makefile.kmk	2013-10-15 23:35:31.335117944 +0300
@@ -683,7 +683,7 @@ $(VBOXWEB_OUT_DIR)/gsoap_generate_all_ts
 : $(VBOXWEB_GSOAPH_FROM_GSOAP) $(VBOXWEB_GSOAPH_FROM_XSLT) $(VBOX_NSMAP) $(RECOMPILE_ON_MAKEFILE_CURRENT) | $$(dir $$@)
 	$(call MSG_GENERATE,,lots of files,$(GSOAPH_RELEVANT))
 	$(RM) -f $@
-	$(REDIRECT) -C $(VBOXWEB_OUT_DIR) -- $(VBOX_SOAPCPP2) $(VBOXWEB_SOAPCPP2_SKIP_FILES) -L -2 -w -I$(VBOX_PATH_GSOAP_IMPORT) $(GSOAPH_RELEVANT)
+	$(REDIRECT) -C $(VBOXWEB_OUT_DIR) -- $(VBOX_SOAPCPP2) $(VBOXWEB_SOAPCPP2_SKIP_FILES) -z1 -L -2 -w -I$(VBOX_PATH_GSOAP_IMPORT) $(GSOAPH_RELEVANT)
 	$(APPEND) $@ done
 
 # copy the generated headers and stuff. This has to be a separate rule if we
